src/test/java/io/confluent/connect/elasticsearch/MappingTest.java

```diff
@@ -109,7 +111,7 @@ private void verifyMapping(Schema schema, JsonObject mapping) throws Exception {
       }
     }
 
-    DataConverter converter = new DataConverter(true);
+    DataConverter converter = new DataConverter(true, BehaviorOnNullValues.FAIL);
```

Comment:
Do we want the current default here (`IGNORE`)?

Reply:
No, this should be `FAIL` since there aren't any apparent cases for passing a null-valued record to the converter in this test.

Reply:
I agree. `null` value is not tested here. Still, since we need to give an argument there, we should rather give the default, than one that would be used less frequently

Reply:
The `DataConverterTest` verifies that `DataConverter` behaves correctly. This test is merely using a `DataConverter` to check the mapping, and it seems like we'd want the test to fail if there are any unexpected problems. How would using a different `BehaviorOnNullValue` affect this test?

Reply:
I don't think it would affect the test, since this test doesn't seem to check the case of record with a `null` value. My point is that we should not depend on testing something "by accident". If we need a test case for `null` we should add one.